app/views/idv/hybrid_handoff/show.html.erb

```diff
@@ -10,7 +10,11 @@
 <% end %>
 
 <%= render PageHeadingComponent.new do %>
-  <%= t('doc_auth.headings.hybrid_handoff') %>
+  <% if IdentityConfig.store.idv_phone_question_a_b_testing[:show_phone_question] == 1 %>
```

Comment:
I wasn't sure if I should test this. I couldn't find a place in the code where conditional text on this page was tested. Since it's simple view logic that should likely be changed again soon, I didn't see it worth constructing a presenter model to test somehow (and we don't have a `hybrid_handoff` model or something to create a presenter around anyway).

But if I'm missing something and this should be tested, I'm happy to add one!

Reply:
I think it'd be good to have a `show.html.erb_spec.rb` file. It should been created previously, but better late than never? I think it'd suffice just to test the bits you're interested in affected by the changes here.

```rb
require 'rails_helper'

RSpec.describe 'idv/hybrid_handoff/show.html.erb' do
  let(:show_phone_question) {}

  before do
    allow(IdentityConfig.store).to receive(:idv_phone_question_a_b_testing).
      and_return(show_phone_question:)
  end

  context 'with shown phone question' do
    let(:show_phone_question) { true }

    it '...' do
      # ...
    end
  end

  context 'without shown phone question' do
    let(:show_phone_question) { false }

    it '...'
      # ...
    end
  end
end
```

Reply:
Thank you for the guidance, and helpful test starter code! I've pushed up a test.